resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -29,21 +29,6 @@
             <stringProp name="Argument.value">443</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="relyingPartyId" elementType="Argument">
-            <stringProp name="Argument.name">relyingPartyId</stringProp>
-            <stringProp name="Argument.value">Bharathi-Inc</stringProp>
-            <stringProp name="Argument.metadata">=</stringProp>
-          </elementProp>
-          <elementProp name="userClaims" elementType="Argument">
```

Comment:
authTypeCallbackUrl - is this used anywhere, if not lets remove

Reply:
Done